src/set_theory/schroeder_bernstein.lean

```diff
@@ -121,6 +127,7 @@ let ⟨f, hf⟩ := axiom_of_choice e in
 
 end wo
 
+/-- The cardinals are totally ordered. -/
```

Comment:
also, why is there not an order instance? another thing worth noting either here or in impl notes (I think this file probably needs a big impl notes section)

Reply:
It's in `set_theory.cardinal`